dom/events/Event-dispatch-click.html

```diff
@@ -209,6 +209,48 @@
   input.dispatchEvent(new MouseEvent('click'));
 }, `disconnected radio should be checked from dispatchEvent(new MouseEvent('click'))`);
 
+test(() => {
+  const input = document.createElement("input");
+  input.type = "checkbox";
+  input.disabled = true;
+  input.dispatchEvent(new MouseEvent("click"));
+  assert_true(input.checked);
```

Comment:
All of the other tests put the asserts inside an onclick handler. That seems pretty unnecessary, and your test seems nice and straightforward... but I would like you to double-check my logic. There's no need for onclick, right?

Reply:
Yes, all browsers mutates the checked state immediately after dispatching a click event so no need to wait for onclick. Do you want me to fix them here?

BTW, disabled checkboxes mutate without emitting onclick, which complicates this more 🤔

Reply:
Nah, we can leave the existing ones as-is. All is well.